v3/plugins/eclipse/gwt/nightly/meta.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 publisher: eclipse
 name: gwt
```

Comment:
Consider renaming it to `gwt-ide` because `eclipse/gwt` says nothing that it is GWT based IDE for Eclipse Che.